README.md

```
@@ -1 +1,321 @@
-AMP-CLIENT
+## 🎸 AMP
+
+밴드 콘서트·페스티벌에서 흩어져 있는 공식 공지와 팬 나눔 공지를 한 번에 모아 보여주는 실시간 공지 서비스
+
+## 🎧 TEAM-DECIBEL
+
+<table>
+  <tr>
+    <td align="center" style="padding:10px">
+      <img src="https://avatars.githubusercontent.com/u/108279922?v=4$0" width="180"/><br/>
```

markdownlint-cli2 (0.18.1)

Images should have alternate text (alt text)

(MD045, no-alt-text)